src/index.html

```diff
@@ -15,8 +15,105 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
+    <header class="header">
+      <div class="container">
+        <div class="header__logo">
+          <a
+            href="index.html"
+            class="logo__link"
+          >
+            <img
+              src="./images/logo.png"
+              alt="logo"
+              class="logo__image"
+            />
+          </a>
+        </div>
+        <nav class="header__nav">
+          <ul class="menu__list">
+            <li class="menu__item">
+              <a
+                href="#"
+                class="menu__link is-active"
+              >
+                Apple
+              </a>
+            </li>
+            <li class="menu__item">
```

Comment:
```suggestion
            <li class="menu__item">
              <a
                href="#"
                class="menu__link is-active"
              >
                Apple
              </a>
            </li>
            
            <li class="menu__item">
```

Add an empty line between multiple sibling blocks of html